packages/quick_actions/quick_actions_ios/lib/messages.g.dart

```diff
@@ -0,0 +1,160 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+// Autogenerated from Pigeon (v11.0.1), do not edit directly.
+// See also: https://pub.dev/packages/pigeon
+// ignore_for_file: public_member_api_docs, non_constant_identifier_names, avoid_as, unused_import, unnecessary_parenthesis, prefer_null_aware_operators, omit_local_variable_types, unused_shown_name, unnecessary_import
+
+import 'dart:async';
+import 'dart:typed_data' show Float64List, Int32List, Int64List, Uint8List;
+
+import 'package:flutter/foundation.dart' show ReadBuffer, WriteBuffer;
+import 'package:flutter/services.dart';
+
+/// Home screen quick-action shortcut item.
+class ShortcutItemMessage {
+  ShortcutItemMessage({
+    required this.type,
+    required this.localizedTitle,
+    this.icon,
+  });
+
+  /// The identifier of this item; should be unique within the app.
+  String type;
+
+  /// Localized title of the item.
+  String localizedTitle;
+
+  /// Name of native resource to be displayed as the icon for this item.
+  String? icon;
+
+  Object encode() {
+    return <Object?>[
+      type,
+      localizedTitle,
+      icon,
+    ];
+  }
+
+  static ShortcutItemMessage decode(Object result) {
+    result as List<Object?>;
+    return ShortcutItemMessage(
+      type: result[0]! as String,
+      localizedTitle: result[1]! as String,
+      icon: result[2] as String?,
+    );
+  }
+}
+
+class _IosQuickActionsApiCodec extends StandardMessageCodec {
+  const _IosQuickActionsApiCodec();
+  @override
+  void writeValue(WriteBuffer buffer, Object? value) {
+    if (value is ShortcutItemMessage) {
+      buffer.putUint8(128);
+      writeValue(buffer, value.encode());
+    } else {
+      super.writeValue(buffer, value);
+    }
+  }
+
+  @override
+  Object? readValueOfType(int type, ReadBuffer buffer) {
+    switch (type) {
+      case 128:
+        return ShortcutItemMessage.decode(readValue(buffer)!);
+      default:
+        return super.readValueOfType(type, buffer);
+    }
+  }
+}
+
+class IosQuickActionsApi {
```

Comment:
Nit: IOS; see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#capitalize-identifiers-consistent-with-their-spelling